data/frame_type.go

```diff
@@ -78,18 +92,66 @@ func FrameTypes() []FrameType {
 	return []FrameType{
 		FrameTypeTimeSeriesWide,
 		FrameTypeTimeSeriesLong,
+		FrameTypeTimeSeriesMulti,
 		FrameTypeTimeSeriesMany,
+
+		FrameTypeNumericWide,
+		FrameTypeNumericLong,
+		FrameTypeNumericMulti,
 	}
 }
 
-// IsTimeSeries checks if the type represents a timeseries
+// IsTimeSeries checks if the FrameType is KindTimeSeries
 func (p FrameType) IsTimeSeries() bool {
 	switch p {
 	case
 		FrameTypeTimeSeriesWide,
 		FrameTypeTimeSeriesLong,
+		FrameTypeTimeSeriesMulti,
 		FrameTypeTimeSeriesMany:
 		return true
 	}
 	return false
 }
+
+// IsNumeric checks if the FrameType is KindNumeric.
+func (p FrameType) IsNumeric() bool {
+	switch p {
+	case
+		FrameTypeNumericWide,
+		FrameTypeNumericLong,
+		FrameTypeNumericMulti:
+		return true
+	}
+	return false
+}
+
+// Kind returns the FrameTypeKind from the FrameType.
+func (p FrameType) Kind() FrameTypeKind {
+	switch {
+	case p.IsTimeSeries():
+		return KindTimeSeries
+	case p.IsNumeric():
+		return KindNumeric
+	default:
+		return KindUnknown
+	}
```

Comment:
Good catch. I'm not sure what they should be. Going with Table, is seems we could say:
 - Kind is Table
 - Format is Table
 - We abbreviate "TableTable" as "Table"

Could do the same for "directory-listing". I guess the assumption here is there will only ever be one format for the kind, but we could do "DirectoryListingSomething" or "TableSomething" if we have to someday. WDYT?

Reply:
I'm also perhaps unsure about "Table" itself.

For example, with a SQL query like like `Select Avg(foo), city group by city`, at least in SSE, it would be looking for "NumericLong", but the in the SQL query UI in grafana that would be format as "Table". And this also makes me realize the collision on the word "format" between the contract and UI.

Reply:
Moved these to https://github.com/grafana/grafana-plugin-sdk-go/issues/572 so can go forward with this PR for now.